web/themes/custom/vote_gov/templates/block/block--system-menu-block--footer.html.twig

```diff
@@ -7,26 +7,50 @@
 
 {% set heading_id = attributes.id ~ '-menu'|clean_id %}
 
-{# footer identifier: agency #}
-{{ 'An official website of the' | t }}
-{{ link('General Services Administration.', 'https://www.gsa.gov/') }}
+  {# {% block contact %}
+    {{ 'Vote.gov in partnership with EAC' | t }}
+    {{ link('Follow Vote.gov on Twitter', 'https://twitter.com/votegov') }}
+  {% endblock %} #}
 
-<nav role="navigation"
-     aria-labelledby="{{ heading_id }}"{{ attributes.addClass(classes)|without('role', 'aria-labelledby') }}>
-  {# Label. If not displayed, we still provide it for screen readers. #}
-  {% if not configuration.label_display %}
-    {% set title_attributes = title_attributes.addClass('visually-hidden') %}
-  {% endif %}
-  {{ title_prefix }}
-  <h2{{ title_attributes.setAttribute('id', heading_id) }}>{{ configuration.label }}</h2>
-  {{ title_suffix }}
-
-  {# Menu. #}
-  {% block content %}
-    {{ content }}
-  {% endblock %}
-</nav>
-
-{#footer identifier:link #}
-{{ 'Looking for U.S. government information and services? ' | t }}
-{{ link('Visit USA.gov', 'https://www.usa.gov/') }}
\ No newline at end of file
+{# identifier start #}
+<div class="usa-identifier">
+<div class="grid-container">
+  <section class="usa-identifier__section usa-identifier__section--masthead" aria-label="Agency identifier,">
```

Comment:
mark aria-label value for string translations like this `{{ 'Agency identifier' | t }}`